src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -18,4 +19,18 @@ browser.runtime.onMessage.addListener((command) => dispatchMap.get(command)?.())
 
 document.addEventListener('keyup', ev => {
     if (ev.key === guard) replaceFuzzyDate();
-});
\ No newline at end of file
+});
+
+
+
+//@ts-ignore couldn't resolve this typing
+browser.runtime.onMessage.addListener((message,sender,sendResponse) => {
+    switch (message) {
```

Comment:
that's what the dispatchMap above is for ;) you just need to add your event and a function there

Reply:
lol you're right. took me so long to figure out the `sendMessage` side that somehow I thought your code was only for key commands